### PR TITLE
feat: [SRM-11310]: add filter on MS page

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by vasuki-kalluri** [comment]
retrigger build

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/components/MonitoredServiceGraphView/MonitoredServiceGraphView.tsx [hunk 1]
```diff
@@ -56,6 +57,7 @@ const ServiceDependencyGraph: React.FC<ServiceDependencyGraphProps> = ({
   } = useGetServiceDependencyGraph({
     queryParams: {
       ...pathParams,
+      filter: search,
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/components/MonitoredServiceGraphView/MonitoredServiceGraphView.tsx:60]: Please add the `filter` query param only if the search exists

**Review by praneshg239 — APPROVED** [review]

**Review by deepesh-ui — APPROVED** [review]

**Comment by vasuki-kalluri** [comment]
trigger cypress

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/components/MonitoredServiceGraphView/MonitoredServiceGraphView.tsx (hunk 1 above)
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/components/MonitoredServiceGraphView/MonitoredServiceGraphView.tsx:60]:
```ts
...(search ? {filter: search}: {})
```

**Pull request merged**